pkg/kubernetes/manifest/manifest.go

```diff
@@ -0,0 +1,155 @@
+package manifest
```

Comment:
There's a lot of undocumented exported stuff here. Could you give them some docs?